setup.py

```diff
@@ -79,6 +79,7 @@
         'nbconvert>=5.6.1,<6.0',
         'nbdime>=3.0.0.b1',
         'nbformat>=5.1.2',
+        'networkx>=2.5.1',
```

Comment:
How is this new dependency being used?

Reply:
I’m guessing this is for the circular references check